lib/workos/user_management.rb

```diff
@@ -748,7 +748,8 @@ def get_organization_membership(id:)
       #
       # @param [Hash] options
       # @option options [String] user_id The ID of the User.
-      # @option options [String] organization_id Filter Users by the organization they are members of.
+      # @option options [String] organization_id Filter memberships by the organization they are members of.
+      # @option options [Array<String>] statuses Filter memberships by status.
```

Comment:
Is `Array<String>` correct here?

Reply:
Given the API accepts an array, I think that's correct